### PR TITLE
Flexible schedule api Documentation

### PR DESCRIPTION
**Pull request by fsi206914**
Close #709 

This pull request is to write API documentation for Cron Schedule.

### CODE REVIEW

**Comment by fsi206914** [comment]
@HappyRay Could you please have a look at this?

File: _includes/docs/latest/ajax-api.html [hunk 1]
```diff
@@ -1130,6 +1128,90 @@ <h3 id="api-schedule-a-flow">Schedule a Flow</h3>
...
+      <td>cronExpression</td>
+      <td>A CRON expression is a string comprising 6 or 7 fields separated by white space[8] that represents a set of times. In azkaban, we use <a href="http://www.quartz-scheduler.org/documentation/quartz-2.x/tutorials/crontrigger.html"> Quartz Cron Format</a>.</td>
+    </tr>
+  </tbody>
+</table>
+
+<p>Here's a curl command sample:</p>
+
+<pre class="code">
+curl -k https://HOST:PORT/schedule -d "ajax=scheduleCronFlow&projectName=PROJECT_NAME&flow=FLOW_NAME&cronExpression=0 * 5,7-10 ? * 4-6" -b azkaban.browser.session.id=SESSION_ID
```
Comment (HappyRay) [inline _includes/docs/latest/ajax-api.html]: Does the expression need to be URL encoded? Do you need to pass the additional parameter? 
see

"To URL-encode the value of a form field you may use --data-urlencode."
https://curl.haxx.se/docs/manpage.html

Comment (fsi206914) [inline _includes/docs/latest/ajax-api.html]: Good Question! Just googled what is URL encoding. We did have a couple of special characters allowed by Quartz. Let me have a try.

Comment (fsi206914) [inline _includes/docs/latest/ajax-api.html]: @HappyRay , I tested some cron expressions, like 0 23/30 5,7-10 ? * 6#3 . All work.

Comment (fsi206914) [inline _includes/docs/latest/ajax-api.html]: Fixed by replacing by --data-urlencode mode

**Pull request merged**